starter/script.js

```diff
@@ -1 +1,45 @@
-// put code here!
+var buffer = function(){
+  console.log("*********************");
+}
```

Comment:
How about combining this and all your other `console.log`s into one function?

Instead of:
```
var buffer = function(){
  console.log("***");
}
console.log("blah");
buffer();
```
...how about:
```
var buffer = function(message){
  console.log("***");
  console.log(message);
}
buffer("blah");
```